site/content/recipes/find_a_reviewer.md

```diff
@@ -0,0 +1,184 @@
+suitable to perform the role of _reviewer_ in the code review.
+
+# Steps
+
+1. Identify the top areas to search for candidate reviewers.
+  - Your research group
+  - Research Software Engineer (RSE) group at your institution
+  - Language or library specific groups at your institution
+  - Open source or research communities you contribute to
+  - Others?
+  - Last resort is to throw it up on Code Review Stack Exchange
+    <https://codereview.stackexchange.com/>
+2. Pre-screen candidates based on your existing knowledge and some criteria
+   (below).
+3. Contact candidates directly or make advertisement.
+4. Repeat 1 to 3 until you get a bite.
+5. Confirm candidate can fulfill minimum thresholds.
+
+# Description
+
+It might seem strange, but code review has something in common with partner
+dancing: it takes two to tango and (at least) two to review code. Without a
+reviewer, the code review cannot happen. And just as a good dance partner is
+essential for a well-performed routine, a suitable reviewer ensures that the
+code review will be a rewarding and valuable experience. Finding someone to
+fill this important role can be one of the more difficult steps in getting a
+code review off the ground — or dance floor, depending on how far you want the
+analogy to go.
+
+Perhaps the main barrier to finding a reviewer is misconceptions about what
+experience a reviewer should possess. As a lone coder, you might be thinking,
+"Only software developers do code review, so I should probably find someone
+outside of research." Or "The reviewer definitely needs to be more experienced
+than me." We want to be absolutely clear on this point: almost _any_ researcher
+who codes can be a good reviewer of your research code.
+
+That being said, there are certainly minimum criteria that a reviewer should
+satisfy to make a code review worthwhile for both parties, and there are some
+places to search for reviewers that have a higher chance of yielding someone
+that fulfills these criteria. Let's look at these now.
+
+## Step 1: Places to Look {#search}
+
+When looking for a reviewer, your first port of call should almost always be
+your own research group. Your colleagues are the most likely to be using the
+same language and libraries as you, and they will be familiar with your
+common research domain. It will be least intimidating to ask someone at the
+same seniority level as you, but equally, you shouldn't feel ashamed or
+intimidated to show your code to someone at a higher seniority. You already
+show them your scientific results after all! However, there are many reasons
+why it may not be possible to seek a reviewer in your research group. Your
+group might be small and you are the only coder, or perhaps the idea of
+reviewing code is perceived as a waste of time by your colleagues, and you are
+hesitant to stick your neck out. If so, keep reading.
+
+<!--
+Probably need to add something about justifying time on code review to
+supervisor?
+--!>
```

Comment:
Yes, but I think we agreed that this would be some sort of cost-benefit analysis on another section.

Reply:
I have noted that we should link to that section once we write it.